portlets/src/main/resources/locale/navigation/portal/administration_en.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2020 eXo Platform SAS.
```

Comment:
- update header
- you should declare the file into https://github.com/Meeds-io/gamification/blob/feature/experience/translations.properties#L26 like done for global site, in order to be considered by crowdin